### ISSUE TITLE
Issue API : Get Time Tracking (Time Stats) information

### ISSUE THREAD

**Issue by jwillp** [m4tthumphrey/php-gitlab-api#176]
Hello,
I was trying to get time stats of issues but I could not seem to find how to do it anywhere in the code.

Adding this to lib\Gitlab\Api\Issues.php does the trick:

```php
    /**
     * @param int $project_id
     * @param int $issue_iid
     * @return mixed
     */
    public function showTimeStats($project_id, $issue_iid)
    {
        return $this->get( $this->getProjectPath($project_id, 'issues/'.$this->encodePath($issue_iid) .'/time_stats'));
    }
```
Note: You need to specify the issue's iid, not the id.

I havent had the time to try something for the Model Class, so I will not make a pull request for the moment.

**Comment by m4tthumphrey** [comment]
This is just something that hasn't been added to the wrapper. Would be great if you could have a go at adding it? 😄

**Comment by TheKvist** [comment]
As I found this a nice feature I thought I'd take care of it. Slightly renamed the proposed method name as the method rather gets the time stats than show it. I think I will add the other time tracking-related methods, too in the next time.

**Comment by fbourigault** [comment]
Fixed by #178.
